### PR TITLE
Update ConfirmLeaveModal.js

### PR DESCRIPTION
**Pull request by Nhamakami7**
I'm not completely sure if my changes from class to functional were correct. Could I have my work checked over?

### CODE REVIEW

File: src/components/Classes/components/ConfirmLeaveModal.js [hunk 1]
```diff
@@ -3,24 +3,27 @@ import ReactModal from "react-modal";
 import { Container, Row, Col, Button } from "reactstrap";
 import * as fetch from "../../../lib/fetch.js";
 
-class ConfirmLeaveModal extends React.Component {
+function ConfirmLeaveModal(props) {
+  const [error, setError] = useState("");
```
Comment (lumisphere902) [inline src/components/Classes/components/ConfirmLeaveModal.js:7]: You need to import `useState` to use it here. For example, `import React, { useState } from "react";`

File: src/components/Classes/components/ConfirmLeaveModal.js [hunk 2]
```diff
@@ -3,24 +3,27 @@ import ReactModal from "react-modal";
...
+  /*
```
Comment (lumisphere902) [inline src/components/Classes/components/ConfirmLeaveModal.js:8]: You can just delete this instead of commenting it out.

**Review by lumisphere902 — CHANGES_REQUESTED** [review]
Mostly good, just a couple of small issues. Also, you should merge in the latest master and fix all the lint issues. Let me know if you need help with that!

**Comment by reginawang3495** [comment]
I'd recommend testing this out!! This probably wouldn't have compiled :)

File: src/components/Classes/components/ConfirmLeaveModal.js [hunk 3]
```diff
@@ -31,64 +31,66 @@ class ConfirmLeaveModal extends React.Component {
         .leaveClass(data)
         .then((res) => {
           if (!res.ok) {
-            this.setState({
-              error: res.error || 'Failed to leave the class, please try again later',
-            });
+            // this.setState({
+            //  error: res.error || "Failed to leave the class, please try again later",
+            // });
```
Comment (reginawang3495) [inline src/components/Classes/components/ConfirmLeaveModal.js:36]: remove?

File: src/components/Classes/components/ConfirmLeaveModal.js [hunk 4]
```diff
@@ -31,64 +31,66 @@ class ConfirmLeaveModal extends React.Component {
...
           }
-          this.props.removeStudentClass(this.props.cid);
-          this.props.inClass ? this.props.unsetClass() : this.closeModal();
+          removeStudentClass(cid);
+          if (inClass) {
+            unsetClass();
+          } else {
+            closeModal();
+          }
+          // props.inClass ? props.unsetClass() : closeModal();
```
Comment (reginawang3495) [inline src/components/Classes/components/ConfirmLeaveModal.js:46]: remove?

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
just some removals :)

**Review by reginawang3495 — APPROVED** [review]
small nits then feel free to merge!

**Pull request merged**